spec/factories/applicant_counsels.rb

```diff
@@ -8,8 +8,20 @@
     lastName { 'Rob' }
     status { 'Random' }
     after(:build) do |applicant_counsel|
-      applicant_counsel.applicants << FactoryBot.build(:applicant)
-      applicant_counsel.attendance_days << FactoryBot.build(:attendance_day)
+      applicant_counsel.applicants << build(:applicant)
+      applicant_counsel.attendance_days << build(:attendance_day)
+    end
+  end
+
+  factory :realistic_applicant_counsel, class: 'ApplicantCounsel' do
+    title { Person::TITLES.sample }
+    firstName { Faker::Name.first_name }
+    middleName { Faker::Name.middle_name }
+    lastName { Faker::Name.last_name }
+    status { Faker::Demographic.race }
```

Comment:
I'm not sure what `status` refers to for a counsel but I suspect it's not their race. Probably ok to leave this for now and change it later if it causes problems.

Reply:
Good point, I will change this one in a follow up PR.

Reply:
Fixed by https://github.com/ministryofjustice/hmcts-common-platform-mock-api/pull/132